Enterprise-CMCS/macpro-onemac
# Add triage choices for new landing pages

Story: https://qmacbis.atlassian.net/browse/OY2-24016

### Details

This PR adds triage options under the Medicaid SPA and CHIP SPA options -- now each of them have a subset of options where one or more option links to a landing page (added in #1250) and one that links to the SPA form itself.

### Changes

- 3 sub choices for Medicaid SPA -- one links to MMDL landing page, one links to MacPRO landing page, one links to Medicaid SPA form
- 2 sub choices for CHIP SPA -- one links to MMDL landing page, one links to the CHIP SPA form

<img width="300" alt="Screen Shot 2023-05-22 at 4 39 30 PM" src="https://github.com/Enterprise-CMCS/macpro-onemac/assets/4022304/45aa0802-5af6-4544-8ef2-72c1fab3be5e">
<img width="300" alt="Screen Shot 2023-05-22 at 4 39 35 PM" src="https://github.com/Enterprise-CMCS/macpro-onemac/assets/4022304/e5c4f8e4-bccd-49f3-9f6c-b42819fdf765">


### Implementation Notes

- Thanks @kristin-at-theta for the clean PR last time we did these -- super easy to add these referencing that PR
- Visited links are highlighted in the default purple -- I think we might wanna consider declaring the visited style with the desired text color, but the requirements did not state anything about this so I let it slide.

### Test Plan

1. Log in as a submitter
2. Navigate to the Dashboard > New Submissions triage page
3. Click the SPA option
4. Click the Medicaid Spa option
5. **Assert new options are present**
6. Select first option and **assert this leads to the correct landing page in accordance with the card description**
7. Click the back arrow
8. Select second option and **assert this leads to the correct landing page in accordance with the card description**
9. Click the back arrow
10. Select the third option and **assert this takes you to the Medicaid SPA form**
11. Click the back arrow and confirm exit
12. Repeat steps 2-3
13. Click the CHIP SPA option
14. Select first option and **assert this leads to the correct landing page in accordance with the card description**
15. Click the back arrow
16. Select second option and **assert this leads to the CHIP SPA form**
